### ISSUE TITLE
erl_terms version is broken with setuptools >= 58

### ISSUE THREAD

**Issue by tyler-jachetta** [otakup0pe/avakas#91]
There should either be a version restriction for setuptools to <58, or the erl_terms dependency must be updated

**Comment by tyler-jachetta** [comment]
this was on py3.9, checking if problem in py3.8 (which is tested)

**Comment by tyler-jachetta** [comment]
yes, confirmed happens in python 3.8 with upgraded (>=58) setuptools

**Comment by otakup0pe** [comment]
It was bound to happen. Might just pull the bespoke erlang support as `erl_terms` ain't been updated in a minute.

**Comment by tyler-jachetta** [comment]
it was in the back of my head, will do
